scripts/set-react-version.js

```diff
@@ -15,7 +15,7 @@ const os = require("os");
 
 const VALID_TAGS = ["canary-macos", "canary-windows", "main", "nightly"];
 const REACT_NATIVE_VERSIONS = {
-  "canary-macos": "^0.64",
+  "canary-macos": "^0.66",
```

Comment:
just a simple question for me to learn - here you can put 0.66 already even if it's not out yet?

Reply:
(as far as I can see, the canary macos on npm have the `0.0.0-hash` format)

Reply:
This is for pointing at the correct version of react-native to install with react-native-macos. The latter doesn't specify which version to use because it is self-reliant, but I think cli complains if it cannot find react-native.